meter.py

```
@@ -0,0 +1,235 @@
+            all_samples[key]["volume_db"] = m.get("volume_db", all_samples[key]["volume_db"])
+
+        if i < num_samples - 1:
+            time.sleep(interval)
+
+    return all_samples
+
+
+def analyze(all_samples):
+    """Compute statistics for each track."""
+    results = []
+    for key, data in all_samples.items():
```

Ruff (0.15.1)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)